tests/integration/components/frost-text-test.js

```diff
@@ -16,29 +19,318 @@ describeComponent(
+
+      expect(
+        this.$('.frost-text-clear'),
+        'class "frost-text-clear" is set'
+      ).to.have.length(1)
+    })
+
+    it('sets align property', function () {
+      const align = 'right'
+
+      this.set('align', align)
+
+      this.render(hbs`
+          {{frost-text
+            align=align
+          }}
+      `)
+
+      expect(
+        this.$('input').hasClass('right'),
+        'class "right" is set'
+      ).to.be.true
+    })
+
+    it('sets autocapitalize property', function () {
+      this.render(hbs`
+          {{frost-text
+            autocapitalize='on'
+          }}
+      `)
+
+      expect(
+        this.$('input').attr('autocapitalize'),
+        'autocapitalize attribute is set'
+      ).to.eql('on')
+    })
+
+    it('sets autocorrect property', function () {
+      this.render(hbs`
+          {{frost-text
+            autocorrect='on'
+          }}
+      `)
+
+      expect(
+        this.$('input').attr('autocorrect'),
+        'autocorrect attribute is set'
+      ).to.eql('on')
+    })
+
+    it('sets autofocus property', function () {
+      const autofocus = 'true'
+
+      this.set('autofocus', autofocus)
+
+      this.render(hbs`
+          {{frost-text
+            autofocus=autofocus
+          }}
+      `)
+
+      expect(
+        this.$('input').attr('autofocus'),
+        'autofocus attribute is set'
+      ).to.eql('autofocus')
+    })
+
+    it('sets maxlength property', function () {
+      const maxlength = '30'
+
+      this.set('maxlength', maxlength)
+
+      this.render(hbs`
+        {{frost-text
+          maxlength=maxlength
+        }}
+     `)
+
+      expect(
+        this.$('input').attr('maxlength'),
+        'maxlength is set'
+      ).to.eql('30')
+    })
+
+    it('sets placeholder property', function () {
+      const placeholder = 'Enter here'
+
+      this.set('placeholder', placeholder)
+
+      this.render(hbs`
+        {{frost-text
+          placeholder=placeholder
+        }}
+     `)
+
+      expect(
+        this.$('input').attr('placeholder'),
+        'placeholder is set'
+      ).to.eql('Enter here')
+    })
+
+    it('sets value property', function () {
+      const value = 'Testing'
+
+      this.set('value', value)
+
+      this.render(hbs`
+        {{frost-text
+          value=value
+        }}
+      `)
+
+      expect(
+        this.$('input').val(),
+        'value is set'
+      ).to.eql('Testing')
+    })
+
+    it('set disabled property', function () {
+      this.render(hbs`
+        {{frost-text
+          disabled=true
+        }}
+      `)
+
+      expect(
+        this.$('input').attr('disabled'),
+        'disabled attribute is set'
+      ).to.eql('disabled')
+    })
+
+    it('set readonly property', function () {
+      this.render(hbs`
+        {{frost-text
+          readonly=true
+        }}
+      `)
+
+      expect(
+        this.$('input').attr('readonly'),
+        'readonly attribute is set'
+      ).to.eql('readonly')
```

Comment:
Does this pass?  I would have expected it to either equal `true` (from above) or null (if the value is stripped)